src/sonic-bgpcfgd/bgpcfgd/managers_device_global.py

```diff
@@ -259,8 +257,8 @@ def downstream_isolate_unisolate(self, idf_isolation_state):
             log_err("IDF: invalid value({}) is provided".format(idf_isolation_state))
             return False
 
-        if self.switch_type and self.switch_type != "SpineRouter":
-            log_debug("DeviceGlobalCfgMgr:: Skipping IDF isolation configuration on Switch type: %s" % self.switch_type)
+        if self.switch_role and self.switch_role != "SpineRouter":
+            log_debug("DeviceGlobalCfgMgr:: Skipping IDF isolation configuration on %s" % self.switch_role)
```

Comment:
Do we need this change in 202311, 202405?

Reply:
needed for 202405, not 202311